plugins/modules/zabbix_service.py

```diff
@@ -228,7 +228,7 @@ def main():
     service = Service(module)
     service_ids = service.get_service_ids(name)
     if service_ids:
-        service_json = service.dump_services(service_ids)
+        service.dump_services(service_ids)
```

Comment:
this whole function call is probably uneccessary

Reply:
I’m going to check this function call necessary or unnecessary after the merge.